.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: rust
+sudo: false
+
+matrix:
+  include:
+  - os: osx
```

Comment:
I think we want to include `- os: linux` as well.

Reply:
The default is linux, this adds an extra build that is run against macOS on top of those. You can see the full build matrix here: https://travis-ci.org/Nemo157/license-exprs/builds/376051360

Reply:
How it actually works, each entry in the `include` array defines one job, the first job here overrides the `os` to be `osx` and uses the default `rust` version, the other 3 jobs each override the `rust` version and use the default `os` of `linux`.